### ISSUE TITLE
Mobile lump of breast

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10068]
**Preferred term label:**
Mobile lump of breast
and it would be good to have the equivalent for immobile too.

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C4028586

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0032408

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fibrocystic breast disease
and for immobile: Mucinous breast carcinoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Breast masses can be assessed as to  size, shape, tenderness, fluctuance, mobility, texture, and pulsatility. I wonder if it wouldn't be better to use modifiers for this?

**Comment by MickeySegal** [comment]
Agree.  Our term is actually a modifier.

**Comment by pnrobinson** [comment]
@MickeySegal 
two new modifier terms:

HP:0025759
Mobile lump

HP:0025760
Immobile lump
